.env.sample

```diff
@@ -1,4 +1,4 @@
 PORT=8000
-DATABASE_URL=postgres://docker:docker@127.0.0.1:5400/api-dev
-OKTA_URL_ISSUER=https://example.okta.com/oauth2/default
-OKTA_CLIENT_ID=example
\ No newline at end of file
+DATABASE_URL="postgres://docker:docker@127.0.0.1:5400/api-dev"
```

Comment:
the `.env.sample` is not supposed to have the actual values. This is use just to show the keys / structure of the env file. the `.env.sample` is pushed to source control but the actual `.env` is pushed to source control.